src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -8540,6 +8541,10 @@ namespace AirflowNetworkBalanceManager {
         static bool errFlag(false);
         Array1D_int NodeConnectionType; // Specifies the type of node connection
         std::string CurrentModuleObject;
+        int BranchNum;
+        int CompNum;
+        int NumOfFans;
+        std::string FanNames;
```

Comment:
We can define these variables where they are used instead of up here at the top.

Reply:
@Myoldmopar I move these variables before their use.